### PR TITLE
Fixed Wifi to WiFi

### PR DESCRIPTION
**Pull request by agdl**

### CODE REVIEW

File: examples/AP_SimpleWebServer/AP_SimpleWebServer.ino [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 int led =  LED_BUILTIN;
 
-char ssid[] = "wifi101-network"; // created AP name
+char ssid[] = "WiFi101-network"; // created AP name
```
Comment (sandeepmistry) [inline examples/AP_SimpleWebServer/AP_SimpleWebServer.ino]: I would leave this one alone

File: examples/WiFiWebClientRepeating/WiFiWebClientRepeating.ino [hunk 2]
```diff
@@ -13,20 +13,20 @@
  modified 13 Jan 2014
  by Federico Vanzati
 
- http://arduino.cc/en/Tutorial/WifiWebClientRepeating
+ http://arduino.cc/en/Tutorial/WiFiWebClientRepeating
```
Comment (sandeepmistry) [inline examples/WiFiWebClientRepeating/WiFiWebClientRepeating.ino]: Will this break the link?

Comment (agdl) [inline examples/WiFiWebClientRepeating/WiFiWebClientRepeating.ino]: Actually the right link is the changed one!

File: examples/AP_SimpleWebServer/AP_SimpleWebServer.ino (hunk 1 above)
Comment (agdl) [inline examples/AP_SimpleWebServer/AP_SimpleWebServer.ino]: ok I will commit it now

Comment (agdl) [inline examples/AP_SimpleWebServer/AP_SimpleWebServer.ino]: commit here https://github.com/agdl/WiFi101/commit/497b12e8eb098cea061677133fb3cf917370602f

**Pull request merged**